examples/electron/package.json

```diff
@@ -3,7 +3,10 @@
   "version": "0.0.0",
   "description": "desktopJS Demo",
   "main": "electron.js",
+  "private": true,
```

Comment:
This is one of the lerna packages so the desktopjs symlink works but we don't want it published.